### ISSUE TITLE
Multiline expressions

### ISSUE THREAD

**Issue by hamishforbes** [vectordotdev/vrl#854]
I'm just getting starting with Vector and I'm struggling with just laying out complex if statements over multiple lines.

This is I guess part doc request, I haven't found anything in the docs that describes how to split expressions over multiple lines
and part feature request to improve the support here

As an example I have a vrl remap to drop noisy/useless log lines
one of the statements in that program is this, which works

```
if .kubernetes.pod_labels."common.k8s.elastic.co/type" == "elasticsearch" &&
  (contains(.message, "cname_in_publish_address") ||
  contains(.message, "Fielddata access on the ")) { abort }
```

but this syntax feels a bit awkward and ideally I'd like to do something like this

```
if .kubernetes.pod_labels."common.k8s.elastic.co/type" == "elasticsearch"
  && (
    contains(.message, "cname_in_publish_address")
    || contains(.message, "Fielddata access on the ")
  )
{ abort }
```

Which is invalid but I don't really understand why.

**Comment by jszwedko** [comment]
Agreed, this should be better documented and probably improved to allow splitting lines in more places.
